Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -265,16 +265,20 @@ Note: The [microsoft-teams-ring-user](https://learn.microsoft.com/en-us/graph/ap
 2. Enter your Client/Application ID in the *Bot ID* parameter. 
 3. Enter your Client Secret in the *Bot Password* parameter.
 4. Save the instance.
+5. [Add the Demisto Bot to a Team](#Add-the-Demisto-Bot-to-a-Team)
```

Comment:
for client credentials flow, no need to add the bot to the team to test the instance, totally fine (and makes sense) and test it right after configuring the instance